code/modules/mob/living/simple_animal/hostile/mothership_creatures.dm

```diff
@@ -264,6 +272,19 @@
 				to_chat(user, "<span class='warning'>[O] is full.</span>")
 			if(!transfered)
 				to_chat(user, "<span class='warning'>[src]'s tendrils are dry. Wait a bit longer...</span>")
+		if(trades_coins == 1 && istype(O, /obj/item/weapon/reagent_containers/food/snacks/zam_notraisins))
```

Comment:
also while you're at it, this is interpreted the same as `if(trades_coins && ...)`
You don't need to compare it to 1, you just need to make sure it's non-zero - this is purely style though